test/resources/add-token-pair/development/WETH_FCR.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  // WETH
+  tokenB: {
+    symbol: 'WETH',
+    address: "0x4cadb4939da3368e89fbf6d42fd3e17811bd35b4",
+    // Check ETH oracle:
+    //   https://makerdao.com/feeds/#0x729d19f657bd0614b4985cf1d82531c67569197b
+    //   Price: 423.960
+    //   20$ = 20/423.960 ETH = 0.04717426172
+    funding: 10
+  },
+  // OMG
```

Comment:
Can change this to 'FCR' and remove the commented out OMG code, I think

Reply:
yep!